automation/src/test/scala/org/broadinstitute/dsde/test/api/TestMethods.scala

```diff
@@ -0,0 +1,194 @@
+package org.broadinstitute.dsde.test.api
+
+import org.broadinstitute.dsde.workbench.fixture.Method
+import org.broadinstitute.dsde.workbench.model.google.{GcsBucketName, GcsObjectName, GcsPath}
+
+trait TestMethods {
+  /* METHODS */
+
+  val SubWorkflowChildMethod = Method(
+    methodName = "child_method_for_subworkflow_test",
+    methodNamespace = "subworkflow_methods",
+    snapshotId = 1,
+    rootEntityType = "participant",
+    synopsis = "testtestsynopsis",
+    documentation = "",
+    payload =
+      """
+        |workflow turboHC {
+        |
+        |	File refFasta
+        |	File refIndex
+        |	File refDict
+        |
+        |	File inputBam
+        |	File inputBamIndex
+        |	File intervalsList
+        |
+        |	String gatk_docker
+        |
+        |	String output_name = basename(inputBam, ".bam") + ".g.vcf"
+        |
+        |	Array[String] callingIntervals = read_lines(intervalsList)
+        |
+        |	scatter(interval in callingIntervals) {
+        |		call HaplotypeCaller_GVCF {
+        |			input:
+        |				refFasta = refFasta,
+        |				refIndex = refIndex,
+        |				refDict = refDict,
+        |
+        |				inputBam = inputBam,
+        |				inputBamIndex = inputBamIndex,
+        |				intervals = interval,
+        |				gvcf_name = output_name,
+        |				docker_image = gatk_docker
+        |		}
+        |	}
+        |	call MergeGVCFs {
+        |		input:
+        |			gvcfs = HaplotypeCaller_GVCF.output_gvcf,
+        |			gvcf_name = output_name,
+        |			docker_image = gatk_docker
+        |	}
+        |
+        |	output {
+        |		File outputGVCF = MergeGVCFs.mergedGVCF
+        |	}
+        |}
+        |
+        |task HaplotypeCaller_GVCF {
+        |
+        |	File refFasta
+        |	File refIndex
+        |	File refDict
+        |	File inputBam
+        |	File inputBamIndex
+        |
+        |	String gvcf_name
+        |	String intervals
+        |
+        |	String docker_image
+        |
+        |	command {
+        |		gatk HaplotypeCaller \
```

Comment:
what's the data we're running this on for the test? does it run in tiny-time? i wonder if hand-rolling an echo-strings wdl would be faster

Reply:
It's pretty small.  Even so, it aborts as soon as it detects that there's subworkflow metadata.